thrust/benchmarks/bench/transform/basic.cu

```diff
@@ -90,3 +93,127 @@ NVBENCH_BENCH_TYPES(basic, NVBENCH_TYPE_AXES(types))
   .set_name("base")
   .set_type_axes_names({"T{ct}"})
   .add_int64_power_of_two_axis("Elements", nvbench::range(16, 28, 4));
+
+namespace babelstream
+{
+// The benchmarks in this namespace are inspired by the BabelStream thrust version:
+// https://github.com/UoB-HPC/BabelStream/blob/main/src/thrust/ThrustStream.cu
+
+constexpr auto startA      = 0.1;
+constexpr auto startB      = 0.2;
+constexpr auto startC      = 0.0;
+constexpr auto startScalar = 0.4;
+
+using element_types    = nvbench::type_list<float, double>;
```

Comment:
remark: I think we should go with your suggestion and extend this with 1, 2, and 16 byte types. I don't have preferences on making those integers or floats, but I think it makes sense to keep float and double here to match babel stream use case.